activity_calendar/templates/activity_calendar/snippet_activity_moment.html

```diff
@@ -0,0 +1,86 @@
+{% load static %}
+{% load calendar_tags %}
+
+
+    <div class="card">
+        <div class="card-img-top bg-secondary" style="height: 8rem;
+            background-image: url({% if activity_moment.promotion_image %}{{ activity_moment.promotion_image.url }}{% endif %});
+            background-size: cover; background-position: 50% 50%"
+        >
+        </div>
+        <div class="card-body">
+            <div class="mb-3">
+                <h4 class="card-title">{{ activity_moment.title }}</h4>
+                <h6 class="card-subtitle mb-2 text-muted">{{ activity_moment.start_date|date:"l j E H:i" }}</h6>
+            </div>
+            {% if activity_moment.location %}
+            <div class="d-flex">
+                <!-- Location Icon -->
+                <div class="col-xs-2">
+                    <img
+                        src="{% static 'octicons/location.svg' %}"
+                        alt="Location-icon"
+                        height="16"
+                        width="16"
+                    />
+                </div>
+                <!-- Location -->
+                <div class="col nospace align-self-center">
+                    <p id="event-location">{{ activity_moment.location }} </p>
+                </div>
+            </div>
+            {% endif %}
+
+            <div class="d-flex">
+                <!-- Subscription Icon -->
+                <div class="col-xs-2">
+                    <img
+                        src="{% static 'octicons/clippy.svg' %}"
+                        alt=""
+                        height="16"
+                        width="16"
+                    />
+                </div>
+                <!-- Subscription status -->
+                <div class="col align-self-center">
+                    <p id="event-status">
+                        {% if user|is_subscribed_to:activity_moment %}
+                            <span class="text-success">
+                                You are subscribed
+                            </span>
+                        {% elif activity_moment.is_open_for_subscriptions %}
+                            {% if activity_moment.get_subscribed_users.count >= activity_moment.max_participants and activity_moment.max_participants != -1 %}
```

Comment:
At some point we might want to make an `ActivityMoment.is_full(..)` method; we're doing this same check in more places